prow/scripts/reconciler-image-autobump.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+
+#Description: Generic image-autobump tool. This scripts implements a pipeline that updates the reconciler docker image tag in kyma-project/control-plane
+#
+#Expected common vars:
+# - JOB_TYPE - set up by prow (presubmit, postsubmit, periodic)
+# - TEST_INFRA_SOURCES_DIR - directory path with kyma-project/test-infra sources
+# - K8S_TEST_INFRA_SOURCES_DIR - directory path with kubernetes/test-infra sources
+# - RECONCILER_DIR - directory path with Kyma-incubator/reconciler sources
+# - CONTROL_PLANE_DIR - directory path with Kyma-project/control-plane sources
+#
+#Please look in each provider script for provider specific requirements
+
+## ---------------------------------------------------------------------------------------
+## Configurations and Variables
+## ---------------------------------------------------------------------------------------
+
+# exit on error, and raise error when variable is not set when used
+set -e
+
+ENABLE_TEST_LOG_COLLECTOR=false
+
+# Exported variables
+export K8S_TEST_INFRA_SOURCES_DIR="/home/prow/go/src/github.com/kubernetes/test-infra"
+export RECONCILER_DIR="/home/prow/go/src/github.com/kyma-incubator/reconciler"
+export CONTROL_PLANE_DIR="/home/prow/go/src/github.com/kyma-project/control-plane"
+export KYMA_TEST_INFRA_SOURCES_DIR="/home/prow/go/src/github.com/kyma-project/test-infra"
+export BUMP_TOOL_CONFIG_FILE="${KYMA_TEST_INFRA_SOURCES_DIR}/prow/scripts/resources/control-plane-autobump-reconciler-config.yaml"
```

Comment:
This `BUMP_TOOL_CONFIG_FILE` need to update after code relocated to reconcile repo